packages/service-library/src/servicelib/secure_random.py

```diff
@@ -0,0 +1,26 @@
+import secrets
+
+
+def secure_randint(start: int, end: int) -> int:
+    """Generate a random integer between start (inclusive) and end (exclusive)."""
+    if not isinstance(start, int) or not isinstance(end, int):
```

Comment:
Why not simply using pydantic's decorator? `validate_arguments`? https://docs.pydantic.dev/1.10/usage/validation_decorator/#validate-without-calling-the-function

You can not only include the type checks but even the `start <= end` constraint

Reply:
thanks for the suggestion.